src/admin/config.yml

```diff
@@ -37,8 +37,36 @@ collections:
           i18n: true,
           required: false,
         }
-      - { label: Navigation Order, name: order, widget: number, i18n: copy }
-      - { label: "Has Banner?", name: hasBanner, widget: boolean, i18n: copy }
+      - label: "Layout"
+        name: layout
+        widget: select
```

Comment:
I feel like providing this layout option in CMS might mislead content manager if they don't know what this field is for. Currently, the options are the layouts we have, but these layouts are unique and specific to a page. When we create a different type of page, resources for an example, we will need to manually add resources layout to this field, and we will be handling adding layout to the new markdown we will be creating for the resources page.

Reply:
The field needs to be there because when creating a translation of the about page (for example) it was not getting added to the page. I've made it hidden and default to the base layout so it isn't editable.